README.md

```diff
@@ -2,8 +2,7 @@
 
 Weave GitOps
 
-[![Coverage Status](https://coveralls.io/repos/github/weaveworks/weave-gitops/badge.svg?branch=main)](https://coveralls.io/github/weaveworks/weave-gitops?branch=main)
-![Test status](https://github.com/weaveworks/weave-gitops/actions/workflows/test.yml/badge.svg)
+![Test status](https://github.com/weaveworks/weave-gitops/actions/workflows/pr.yaml/badge.svg)
```

Comment:
should this not be `[![Test status](https://github.com/weaveworks/weave-gitops/actions/workflows/pr.yaml/badge.svg)`

Reply:
ignore, im being tired and dumb